Makefile

```diff
@@ -1,11 +1,18 @@
+wrangle_data:
+	docker build -t timing_comparison .
+	docker run -it -v $(shell pwd):/home/jovyan/work --rm timing_comparison python -m lib.data.wrangler
```

Comment:
docker run -it -v $(shell pwd):/home/jovyan/work --rm `jupyter/scipy-notebook` python -m lib.data.wrangler